### PR TITLE
solution

### PR DESCRIPTION
**Pull request by illiashcherbak**
[DEMO LINK](https://illiashcherbak.github.io/react_clock/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,39 +1,94 @@
...
-};
+  leftClickHandler = () => {
+    this.setState({ hasClock: true });
+  };
+
+  rightClickHandler = (event: MouseEvent) => {
+    event.preventDefault();
+
+    this.setState({ hasClock: false });
+  };
```
Comment (oleh-yablunovskyi) [inline src/App.tsx:55]: p. 3 of checklist: 
"Method's name should start with a verb (so you could easily tell what your method actually do)"

**Review by oleh-yablunovskyi — CHANGES_REQUESTED** [review]
Please fix the names of handler methods. Otherwise good job!

**Review by oleh-yablunovskyi — APPROVED** [review]

File: src/App.tsx [hunk 2]
```diff
@@ -1,39 +1,94 @@
...
+            '&:hover': {
+              backgroundColor: '#eaa',
+            },
+          }}
+        >
+          <div className="App">
+            <h1>React clock</h1>
+
+            {hasClock && <Clock clockName={clockName} />}
+            {!hasClock && (
```
Comment (ivan-kadykalo) [inline src/App.tsx:84]: Don't forget about breaks between elements
```suggestion
            {hasClock && <Clock clockName={clockName} />}
            
            {!hasClock && (
```

**Review by ivan-kadykalo — APPROVED** [review]
Good job 🔥

**Pull request closed without merging**